servers/httpnodeinfo/server.go

```diff
@@ -70,14 +70,39 @@ func viewHandler(w http.ResponseWriter, r *http.Request) {
 		})
 	}
 	BestBlock, _ := chain.DefaultLedger.Blockchain.GetBlockByHash(chain.DefaultLedger.Blockchain.CurrentBlockHash())
+	var httpRestPort = config.Parameters.HttpRestPort
```

Comment:
Did you tested the code, as I known the modifications here is unnecessary, because the `config.go` in `main` package has done fill default values.